src/routes/settings/assistants/[assistantId]/ReportModal.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
```

Comment:
up to you. Instead of creating a new component, we can just use [prompt](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt)

Reply:
Ha, had I known about this, I think we could have gone for `prompt`, but now that the component is here, might as well use it I guess ? :laughing:

Reply:
(fwiw on the Hub we use window.prompt)

Reply:
(but up to you)

Reply:
then let's do it for consistency, will update :+1:

Reply:
honestly your modal looks cool, let's use it IMO!

Reply:
especially if we are not using`textarea` & want to [keep the text short](https://github.com/huggingface/chat-ui/pull/825#discussion_r1490655236), prompt could be a good choice since we don't have to develop/maintain a component that is similar to native one

Reply:
conflicting opinions. So yeah, Nathan, up to you :)